hydrolib/core/dflowfm/tim/parser.py

```diff
@@ -0,0 +1,91 @@
+import re
+from pathlib import Path
+from typing import List
+
+timpattern = re.compile(r"\s+")
+
+
+class TimTimeSerie:
```

Comment:
Serie is not the singular form of series. Maybe rename it to `TimTimeData` or `TimTimeSeriesRow` ?